File: BocetoV2/Clases/Transaccion.java
using System;

public class Transaccion
{
    public string Descripcion { get; set; }
    public double Cantidad { get; set; }
    public DateTime Fecha { get; set; }
    public TipoTransaccion Tipo { get; set; }

    public Transaccion(string descripcion, double cantidad, DateTime fecha, TipoTransaccion tipo)
    {
        Descripcion = descripcion;
        Cantidad = cantidad;
        Fecha = fecha;
        Tipo = tipo;
    }

    public override string ToString()
    {
        return $"{Fecha.ToShortDateString()} - {Tipo}: {Descripcion} - ${Cantidad:F2}";
    }
}

File: App/AppV1/app/src/main/java/com/pmdm/appV1/models/Transaccion.kt
package com.pmdm.appV1.models

import java.time.LocalDateTime

data class Transaccion(
    var descripcion: String = "",
    var cantidad: Double = 0.0,
    var fecha: LocalDateTime = LocalDateTime.now(),
    var tipo: TipoTransaccion? = null // Enum TipoTransaccion
)
